packages/@adobe/spectrum-css-temp/components/textfield/index.css

```diff
@@ -116,6 +116,12 @@ governing permissions and limitations under the License.
        font-style: normal;
     }
   }
+  /* placeholder gets clipped for arabic script in Chrome */
+  &:lang(ar) {
+    &::placeholder {
+      text-indent: 1px;
```

Comment:
This seemed weird to me. I assumed the issue was because it is a cursive font and it leaned outside of the assumed pixel space for a letter, but it isn't. I checked an English font without cursive, set to RTL and it has this spacing issue when the placeholder ended with a `T`. When I used a cursive English font ending with something like a `T` more then one pixel was hidden. I'm not sure we can assume this 1px will work for all placeholder values in an RTL language.
![image](https://user-images.githubusercontent.com/309540/104534169-9b2a6880-55c8-11eb-80a1-8af98bfe6655.png)
I'm wondering if we should float the placeholder higher up the layer stack to make sure it isn't behind the margin or padding that is covering it.

Reply:
`::placeholder` unfortunately has a pretty limited set of allowed css properties. https://developer.mozilla.org/en-US/docs/Web/CSS/::placeholder
From reading this, I'm not even sure we should be using text-indent at all. I wonder if this happens in the native control as is. I'll give it a try tomorrow

Reply:
I did more research on this. The reason this is happening is because using 'italics' for the font-style applies a ~30 degree lean to the font, it's not actually using an italics font. As a result, it sticks out of the bounds that the actual font takes up. The only way to deal with it is to use text-indent from what I've found.
https://stackoverflow.com/questions/38149704/why-does-the-text-inside-an-input-tag-get-cut-off-even-if-theres-already-a-pa/38150295

Reply:
So should the override be to not apply italics for the `ar` locale if there aren't actually italics in the font?

Reply:
I hadn't thought about that, but maybe. I'll give it a go on Monday

Reply:
Firefox and Safari support the [font-synthesis](https://developer.mozilla.org/en-US/docs/Web/CSS/font-synthesis) property which you can set to `none` to disable fake italics and bold. Unfortunately, not supported in Chrome. https://bugs.chromium.org/p/chromium/issues/detail?id=509989

Reply:
font-synthesis worked great, I'm including it for everything we don't know about. for the specific cases that we know about, I'm fixing it for chrome using font-style: none
Once chrome supports font-synthesis, we can remove that block